src/Tizen.Security.SecureRepository/Tizen.Security.SecureRepository/CertificateManager.cs

```diff
@@ -68,6 +68,37 @@ static public Certificate Get(string alias, string password)
             }
         }
 
+        /// <summary>
+        /// Checks for alias existence
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="alias">The name of a certificate to retrieve.</param>
+        /// <returns>Boolean indicating the exitsance of the alias.</returns>
+        /// <exception cref="ArgumentNullException">
+        /// The alias argument is null.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// Indicates failure to communicate with the Keystore. Check the errorcode for details
```

Comment:
It's not necessarily a communication error. It could be CKMC_ERROR_DB_LOCKED, CKMC_DB_ERROR, CKMC_ERROR_PERMISSION_DENIED or CKMC_ERROR_SOCKET. "Failed to determine the alias existence" sounds good.

Reply:
Are you okay with 'Failed to determine the alias existence. Check the errorcode for details'?

Reply:
Yes, I'm ok with it.